test_config.js

```diff
@@ -0,0 +1,10 @@
+const frisby = require('frisby');
+frisby.globalSetup({
+  request: {
+    headers: {
```

Comment:
This values should be settable... 

I should be able to run all this directly without copy/paste token...

It should be settable by login endpoint